src/Sylius/Component/Promotion/Repository/PromotionCouponRepositoryInterface.php

```diff
@@ -43,4 +43,11 @@ public function countByCodeLength(int $codeLength): int;
      * @return PromotionCouponInterface|null
      */
     public function findOneByCodeAndPromotionCode(string $code, string $promotionCode): ?PromotionCouponInterface;
+
+    /**
+     * @param string $promotionCode
+     *
+     * @return iterable
+     */
+    public function findByPromotionCode(string $promotionCode): iterable;
```

Comment:
Anyway, it's a BC break and should be mentioned in` UPGRADE.md`.

Reply:
good catch, thanks @pamil